elks/fs/msdos/dir.c

```diff
@@ -242,33 +227,10 @@ int fat_readdirx(
 			oldpos = filp->f_pos;
 		}
 		ino = msdos_get_entry(inode,&filp->f_pos,&bh,&de);
-		
-	} //while
+
+	} /* while (ino != -1) */
 
 	if (bh)
 		unmap_brelse(bh);
 	return 0;
 }
-
-static int fat_filldir(
```

Comment:
Always glad to see wasteful wrapper functions go away

Reply:
Yeah, @lithoxs would be happy to see this, as it saves bytes :smile:.